packages/three-vrm-materials-mtoon/src/nodes/MToonNodeMaterial.ts

```diff
@@ -376,12 +380,17 @@ export class MToonNodeMaterial extends THREE.NodeMaterial {
 
       if (this.outlineWidthMode === MToonMaterialOutlineWidthMode.WorldCoordinates) {
         // See about the type assertion: https://github.com/three-types/three-ts-types/pull/1123
-        this.positionNode = (this.positionNode as THREE.ShaderNodeObject<THREE.Node>).add(outlineOffset);
+        this.positionNode = (this.positionNode as ShaderNodeObject<THREE.Node>).add(outlineOffset);
       } else if (this.outlineWidthMode === MToonMaterialOutlineWidthMode.ScreenCoordinates) {
         const clipScale = cameraProjectionMatrix.element(1).element(1);
 
+        // We can't use `positionView` in `setupPosition`
+        // because using `positionView` here will make it calculate the `positionView` earlier
+        // and it won't be calculated again after setting the `positionNode`
+        const tempPositionView = modelViewMatrix.mul(positionLocal);
+
         // See about the type assertion: https://github.com/three-types/three-ts-types/pull/1123
-        this.positionNode = (this.positionNode as THREE.ShaderNodeObject<THREE.Node>).add(
+        this.positionNode = (this.positionNode as ShaderNodeObject<THREE.Node>).add(
           outlineOffset.div(clipScale).mul(positionView.z.negate()),
```

Comment:
It seems the screen-space outline isn't working in three-vrm-materials-mtoon/examples/webgpu-feature-test.

feature-test
![image](https://github.com/user-attachments/assets/b55785fd-110e-42e1-8fb4-c225793989b5)


webgpu-feature-test
![image](https://github.com/user-attachments/assets/ebc794b5-984e-410c-a49b-75cfdaac8a0e)


I think this change is necessary.
```suggestion
          outlineOffset.div(clipScale).mul(tempPositionView.z.negate()),
```

Reply:
Oh I'm sorry, I intended to change this line. Applied it

Reply:
Thanks!